generators/base/generator.mts

```diff
@@ -97,6 +223,20 @@ export default class BaseGenerator extends JHipsterBaseBlueprintGenerator {
     return super.usage().replace('yo jhipster:', 'jhipster ');
   }
 
+  /**
+   * Check if the JHipster version used to generate an existing project is less than the passed version argument
+   *
+   * @param {string} version - A valid semver version string
+   */
+  isJhipsterVersionLessThan(version) {
```

Comment:
In Support?

Reply:
This method has 4 lines and the logic is really simple.
It is used a lot by cleanup tasks.
IMO it's much simpler with the implementation in the class.